ans.txt

```diff
@@ -0,0 +1,6 @@
+1. I think that the table stores information about: the number of the word, the direction, start and end coordinates, the prompt and the answer.
+2.I don't think that the user's inputs are stored in a table because every time the user makes a chenge the table needs to be updated,
+ and I think it will be easier to just store that info in some data structure.
+
+the commend i used to create the table:
+create table tbl(number int, direction text, start_coordinates text, end_coordinates text, prompt text, answer text);
```

Comment:
What about multiple crosswords? you describe only a single crossword and there are multiple ones

Reply:
I really think I lack some fundamental knowledge relating to the question because I can't think of any better solution then to create a new table for every crossword. I understand that a more reasonable and practical solution ought to exist and I know me idea is extremely faulty because it is computationally heavy and redundant but that is the best I have to offer.